src/styles/generic/_layout.scss

```diff
@@ -180,4 +184,8 @@ input, textarea {
   color: #000;
   resize: none;
   font-size: 16px;
+}
+
+#counter {
```

Comment:
Geen styling op ID's als het niet nodig is. Hier kan je ook makkelijk een class gebruiken. Je kan evt. ook een generieke bold class maken die je font-weight bold mee geeft.